### ISSUE TITLE
Magnus effect curving

### ISSUE THREAD

**Issue by Bobobalink** [Chantilly612Code/612-2016#150]
The Magnus effect causes flying objects to curve based on them spinning.
This is a brilliant idea, except that the reaction is nearly impossible to accurately predict because of imperfections with the ball.
If we run out of code to actually write, we could play around and see how plausible this is. Until then, this is on hold.

**Comment by Mister-GoodwinLu** [comment]
well, online research would find the following:
P+(1/2)dv^2= C   ......(eq.1)

In solving for P we get:
  
P=C-(1/2)dv^2   ......(eq.2)

Where

P =Pressure

d=The flowing fluid of constant density

v= Velocity of the fluid

C= constant

and
Fm = S (w × v)
Where:
Fm =the Magnus force vector
w= angular velocity vector of the object
v=Velocity of the fluid (or velocity of object, depends on perspective)

S= air resistance coefficient across the surface of the object

We just need to make sure the pressure/force applied is constant and easy to measure. It's not that difficult. source: http://ffden-2.phys.uaf.edu/211_fall2010.web.dir/Patrick_Brandon/what_is_the_magnus_effect.html

**Comment by Bobobalink** [comment]
Our problem is S, the air resistance coefficient. That is going to vary wildly from ball to ball, and even by orientation. We're never going to be able to predict or measure that with enough certainty to actually  use this effect to do anything useful.

**Comment by WardBenjamin** [comment]
Technically, that actually isn't true. A team that Travis can tell you about did this extremely successfully in (I think) 2014's game.

**Comment by Bobobalink** [comment]
What were the balls like in the 2014 game? Have you seen our reference boulder? It's a mess, with holes and stuff all over it. If we have time, I guess we could test how much that affects the curve, but I hypothesize that will be too much.

**Comment by Alexbay218** [comment]
I talked to multiple people (week 1-3), and everyone said it was unfeasible. This idea was tested by mechanical on several occasions. This is essentially last in priority.

**Comment by WardBenjamin** [comment]
@Alexbay218 Not only are some teams working on this (not saying that we have to), even if it's last on the list there's no reason to get rid of it. Even if this doesn't happen until after competition or at all, it should probably stay on the list.

**Comment by ZackAlfakir** [comment]
My two cents: This would only work if you have a **VERY** _CONSISTANT_ shooter. If you fire the same shot twice, it needs to be the **EXACT** same shot. If it isn't then I don't think it will be possible due to hardware constraints.

**Comment by WardBenjamin** [comment]
@ZackAlfakir I completely agree. We definitely need to get the hall effect sensors and the PID control for the shooter working and tuned correctly before this is even a possibility.

**Comment by ChampD** [comment]
Considering Hall effect sensors don't even work properly, we simply aren't going to do this.

**Comment by WardBenjamin** [comment]
Considering that we still should do this after the season when we do more sensor work, reopening.
